mixer/pkg/runtime/config/snapshot.go

```diff
@@ -44,10 +47,27 @@ type (
 		//  Adapters2 contains adapter metadata loaded from the store
```

Comment:
I am trying to understand why we have the things with the "2" suffix. Are these going to replace the non-suffixed ones in the future? If so, we should add defaults for those.

If some of these are going to be living side-by-side (i.e. Adapters & Templates), then we should use better naming.

Reply:
In my opinion the 2 suffix will replace the old ones. @ozevren what do you mean by `add defaults for those`, not sure I follow ?